src/test/java/org/reaktivity/reaktor/test/internal/k3po/ext/behavior/NukleusBehaviorSystem.java

```diff
@@ -223,6 +241,26 @@ public WriteAdvisedFactory writeAdvisedFactory(
         return writeAdvisedFactories.get(advisoryType);
     }
 
+    private static ChannelHandler newReadFlagsHandler(
+        AstReadOptionNode node)
+    {
+        AstValue<?> flagsValue = node.getOptionValue();
+        int value = flagsValue.accept(new GenerateFlagsOptionValueVisitor(), null);
+        ReadFlagsOptionHandler handler = new ReadFlagsOptionHandler(value);
+        handler.setRegionInfo(node.getRegionInfo());
+        return handler;
+    }
+
+    private static ChannelHandler newWriteFlagsHandler(
+        AstWriteOptionNode node)
+    {
+        AstValue<?> flagsValue = node.getOptionValue();
+        int value = flagsValue.accept(new GenerateFlagsOptionValueVisitor(), null);
```

Comment:
I think this would NPE when it attempts to unbox from `Integer` to `int` if we trigger a codepath that returns `null` from the visitor.